javascript/ql/lib/semmle/javascript/frameworks/helmet/Helmet.Required.Setting.model.yml

```diff
@@ -0,0 +1,7 @@
+extensions:
```

Comment:
It's already included by default, since it is under `frameworks` - there are other models already there, so it just gets picked up along with those that are already referenced